packages/react/src/auto/AutoForm.ts

```diff
@@ -33,6 +33,8 @@ export type AutoFormProps<
   submitLabel?: ReactNode;
   /** What to show the user once the form has been submitted successfully */
   successContent?: ReactNode;
+  /** Called when the form submission completes successfully on the backend */
+  onSuccess?: (record: UseActionFormHookStateData<ActionFunc>) => void;
```

Comment:
This `UseActionFormHookStateData<ActionFunc>` type can make the record not become an `any` type. Example:
![CleanShot 2024-07-15 at 12 45 12@2x](https://github.com/user-attachments/assets/f1bbcc49-95e5-4c85-b79a-0cd6b6e41bf8)